### DISCUSSION TITLE
SPF Record Check not working with multiple SMTP providers [Issue with multiple items in DNS TXT SPF record Check]

### DISCUSSION THREAD

**Discussion by leoncool** [postalserver/postal#3008] [Help with using Postal]
Using multiple SMTP providers is common, in case any SMTP goes down or has an issue.  If an email domain uses multiple SMTP providers, in my case both Mailgun and Postal.  I would have to insert multiple "include" items for TXT DNS record. 
Such as: 
`v=spf1 include:mailgun.org include:spf.MYDOMAIN.com ~all`

Issues: 

1. Postal will not accept this, it keeps showing "No SPF record exists for this domain".  If the SPF checks fails, the "DKIM Record" section will NOT work either.  
2. There is issue with DKIM if the SPF check fails.  Emails will be sent using "s=postal", it should be "postal-jU41dp" in my case. 
Example headers of email sent: 
`
v=1; a=rsa-sha256; c=relaxed/relaxed; d=rp.MYDOMAIN.com; s=postal; t=1718545679; bh=hYItP7NurVhVzkP3RFZRGgILj1Xi4nsyC1aUf6MI7x4=; h=date:from:to:message-id:subject:mime-version:content-type:content-transfer-encoding; b=ForR6fTwwsXRBAN5oI5MzY4p3yi654VO6dTn/Vn2w8Wf/tOKbw+tvz0O3LknNSBBNekni0g7 iiOQRh8OUO/EAQZgTeF9CD/xPR9/S+E2KxD2vDIfyeC/N0kIgAM+RaCA0k4zM033MQS6lO3Q 9198QEJNu9zlT/xZ1BKoWoM8XCg=`

## Screenshots
![20240617132134](https://github.com/postalserver/postal/assets/2077432/1a28691d-dfd6-4c69-bc43-aec5f37b410b)

**Comment by willpower232** [comment]
Doing a quick `dig TXT email.yourdomainname.com` (you missed one pixelation) suggests you have put up a CNAME for `email.yourdomainname.com` so all other records for that subdomain are ignored.

**Comment by neskhan** [comment]
I have a issue where postal will not accept DNS records as below, 

Please advise.

![PostalDNSissue](https://github.com/user-attachments/assets/4dcb1700-aed1-4560-8705-d9c51f81b521)
